js/views/chatHeadsVw.js

```diff
@@ -18,6 +18,18 @@ module.exports = baseVw.extend({
         this.createChatHead(md).render().el
       );
     });
+
+    this.$chatHeadsContainer = $('.chatConversationHeads');
+    this.$headContainer = $('<div />');
+
+    this.showPerScroll = 10;
+    this.nextChatToShow = 0;
+
+    //listen to scrolling on container
+    this.scrollHandler = __.bind(
+        __.throttle(this.onScroll, 100), this
+    );
+    this.$chatHeadsContainer.on('scroll', this.scrollHandler);
```

Comment:
My fault, I had a remove function in it that did exactly that, and took it out because I figured the remove function in the base model would take care of it. I'll add it back in.